steps/codified-security-bitrise/1.0.0/step.yml

```diff
@@ -0,0 +1,53 @@
+title: Codified Security
+summary: This step uploads your app to Codified Security for automated mobile app
+  security testing
+description: This step uploads your app to Codified Security for automated mobile
+  app security testing
+website: https://codifiedsecurity.com
```

Comment:
`website` should point to the step's website (most of the times, it's github url), please mention the Codified Security's website in step's `description` instead.